src/OmniSharp.Cake/Services/RequestHandlers/Navigation/FindSymbolsHandler.cs

```diff
@@ -26,8 +26,9 @@ public override Task<QuickFixResponse> Handle(FindSymbolsRequest request)
                 return Task.FromResult(new QuickFixResponse { QuickFixes = Array.Empty<QuickFix>() });
             }
 
+            var symbolFilter = (Microsoft.CodeAnalysis.SymbolFilter)(request?.SymbolFilter ?? OmniSharpSymbolFilter.TypeAndMember);
```

Comment:
you should use the same pattern as below `(request?.SymbolFilter).GetValueOrDefault(OmniSharpSymbolFilter.TypeAndMember)` for consistency

Reply:
When i use `using Microsoft.CodeAnalysis;` there will be a Type ambigutity. I could change it to: `using SymbolFilter = Microsoft.CodeAnalysis.SymbolFilter` if you prefer this.